online_meetings/report.py

```diff
@@ -29,18 +31,18 @@
 
 logger.info(ENV)
 
-headers = {
-  "Authorization": f"Bearer {ENV['ZOOM_TOKEN']}"
-}
-
-ZOOM_BASE_URL = ENV.get('ZOOM_BASE_URL', "")
 DEFAULT_SLEEP_TIME = ENV.get('DEFAULT_SLEEP_TIME', 10)
 
 
-def get_request_retry(url: str, headers: Dict[str, Union[str, int]],
+def requests_zoom_auth(url: str, auth: AuthBase,
+                       params: Dict[str, Union[str, int]]) -> requests.Response:
+    return requests.request("GET", url, params=params, auth=auth)
```

Comment:
This is okay, though I'm not sure there's a lot gained by extracting as a function.

Reply:
Yeah, probably not anymore, I used to have not have "auth" but instead a method that generates this class. I realized that was wrong. I'll revert this separate function as it is unnecessary now that auth is passed in.